tests/privacy_simple_network.py

```diff
@@ -0,0 +1,280 @@
+pnodes=args.p
+apiNodes=0        # minimum number of apiNodes that will be used in this test
+# bios is also treated as an API node, but doesn't count against totalnodes count
+minTotalNodes=pnodes+apiNodes
+totalNodes=args.n if args.n >= minTotalNodes else minTotalNodes
+if totalNodes > minTotalNodes:
+    apiNodes += totalNodes - minTotalNodes
+
+Utils.Debug=args.v
+dumpErrorDetails=args.dump_error_details
+keepLogs=args.keep_logs
+dontKill=args.leave_running
+onlyBios=False
+killAll=args.clean_run
+sanityTest=args.sanity_test
+walletPort=args.wallet_port
+
+cluster=Cluster(host=TestHelper.LOCAL_HOST, port=port, walletd=True)
+walletMgr=WalletMgr(True, port=walletPort)
+testSuccessful=False
+killEosInstances=not dontKill
+killWallet=not dontKill
+dontBootstrap=sanityTest # intent is to limit the scope of the sanity test to just verifying that nodes can be started
+
+WalletdName=Utils.EosWalletName
+ClientName="cleos"
+timeout = .5 * 12 * pnodes + 60 # time for finalization with 1 producer + 60 seconds padding
+Utils.setIrreversibleTimeout(timeout)
+
+try:
+    TestHelper.printSystemInfo("BEGIN")
+    cluster.setWalletMgr(walletMgr)
+    Print("SERVER: {}".format(TestHelper.LOCAL_HOST))
+    Print("PORT: {}".format(port))
+
+    cluster.killall(allInstances=killAll)
+    cluster.cleanup()
+    Print("Stand up cluster")
+
+    if cluster.launch(pnodes=pnodes, totalNodes=totalNodes, prodCount=1, onlyBios=False, dontBootstrap=dontBootstrap, configSecurityGroup=True) is False:
+        cmdError("launcher")
+        errorExit("Failed to stand up eos cluster.")
+
+    Print("Validating system accounts after bootstrap")
+    cluster.validateAccounts(None)
+
+    def getNodes():
+        global cluster
+        nodes = [cluster.biosNode]
+        nodes.extend(cluster.getNodes())
+        return nodes
+
+#    cluster.biosNode.kill(signal.SIGTERM)
+
+    def reportInfo():
+        Utils.Print("\n\n\n\n\nInfo:")
+        for node in getNodes():
+            Utils.Print("Info: {}".format(json.dumps(node.getInfo(), indent=4, sort_keys=True)))
+
+    Utils.Print("\n\n\n\n\nCheck after KILL:")
+    Utils.Print("\n\n\n\n\nNext Round of Info:")
+    reportInfo()
+
+    producers = [cluster.getNode(x) for x in range(pnodes) ]
+    apiNodes = [cluster.getNode(x) for x in range(pnodes, totalNodes)]
+    apiNodes.append(cluster.biosNode)
+
+    def createAccount(newAcc):
+        producers[0].createInitializeAccount(newAcc, cluster.eosioAccount)
+        ignWallet = cluster.walletMgr.create("ignition")  # will actually just look up the wallet
+        cluster.walletMgr.importKey(newAcc, ignWallet)
+
+    # numAccounts = 4
+    # testAccounts = Cluster.createAccountKeys(numAccounts)
+    # accountPrefix = "testaccount"
+    # for i in range(numAccounts):
+    #     testAccount = testAccounts[i]
+    #     testAccount.name  = accountPrefix + str(i + 1)
+    #     createAccount(testAccount)
+
+    blockProducer = None
+
+    def verifyInSync(producerNum):
```

Comment:
this function seems to be generic and I see you are using it in `privacy_startup_network.py` . What about moving it to `Cluster.py`

Reply:
Moved along with some other methods